src/jquery.autofilter.js

```diff
@@ -63,15 +63,23 @@
         table._allColumnFilters = allColumnFilters;
     }
 
+    //Creating custom case insensitive expression for contains
+    jQuery.expr[':'].icontains = function(a, i, m) {
+        return jQuery(a).text().toUpperCase().indexOf(m[3].toUpperCase()) >= 0;
+    };
+
     $.fn.autofilter = function() {
         // setup the filter modal dialog
-        var modalWidgetId = "_jquery_autofilter_modal";
-        var modalWidget = $("#" + modalWidgetId);
+        var modalWidgetId = "_jquery_autofilter_modal",
+            modalWidget = $("#" + modalWidgetId),
+            $this = $(this),
+            $inputText;
+
         if (modalWidget.length == 0) { // not created yet
             var modalWidgetCode = "" +
 "<div style='display: none'>" +
     "<div id='" + modalWidgetId + "' class='modal'>" +
-    "    <span class='filter-button-reset'>Select all</span> - <span class='filter-button-reset'>Clear</span>" +
+    "    <span class='all filter-button-reset'>Select all</span> - <span class='clear filter-button-reset'>Clear</span>" +
```

Comment:
This is fine, but since the modal should be unique to the page, we could also use autofiler-like ids (ex. _jquery_modal_select_all)--especially since "clear" and "all" is pretty generic and could easily pick up CSS attributes outside of the library.